src/Entities/ArrowEntity.cpp

```diff
@@ -194,6 +194,10 @@ void cArrowEntity::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 			m_IsInGround = false;  // Yes, begin simulating physics again
 		}
 	}
+	else if (IsInWater())  // Arrow in water?
+	{
+		ApplyFriction(m_Speed, ARROW_WATER_FRICTION, static_cast<float>(a_Dt.count()));  // Yes, slow down arrow
+	}
```

Comment:
Is this just some weird formatting from the web interface?

Reply:
I'm sorry but i don't see any problems. It looked fine on vscode and the CheckStyle script said that everything is okay.

Reply:
I don't see anything wrong here. Were you talking about the line-wrap?

Reply:
For me the indentation is off. But only in the web interface.